{"name": "test_add[failed]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2.5 == 2.6", "trace": "self = <calculator.testcase.test_calc_case.TestCalc object at 0x000001B38534CFD0>, get_calc = <calculator.python_code.calc.Calculator object at 0x000001B385325A00>\nget_add_datas = [1.2, 1.3, 2.6]\n\n    @allure.story(\"加法测试案例\")\n    @pytest.mark.run(order=1)\n    # @pytest.mark.first\n    def test_add(self, get_calc, get_add_datas):\n        \"\"\"\n        get_calc实例化计算器类\n        get_add_datas:获取测试数据\n        \"\"\"\n        with allure.step(\"计算两个数的和\"):\n            # 获取yaml文件内参数\n            result = get_calc.add(get_add_datas[0], get_add_datas[1])\n            # 如果相加数为float型 ，保留两位小数\n        if isinstance(result, float):\n            result = round(result, 2)\n            # 结果添加断言\n>       assert result == get_add_datas[2]\nE       assert 2.5 == 2.6\n\ntest_calc_case.py:29: AssertionError"}, "description": "\n        get_calc实例化计算器类\n        get_add_datas:获取测试数据\n        ", "steps": [{"name": "计算两个数的和", "status": "passed", "start": 1612516764272, "stop": 1612516764272}], "attachments": [{"name": "stdout", "source": "c5a3f803-5ab0-4ee2-875f-5097aa70241e-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_add_datas", "value": "[1.2, 1.3, 2.6]"}], "start": 1612516764272, "stop": 1612516764273, "uuid": "fe8287bd-408f-42a6-978d-bd2374e762aa", "historyId": "a7f7c215df765f0ae6544fc6b2952bab", "testCaseId": "22447275a8299bb530d01147a19d08fb", "fullName": "test_calc_case.TestCalc#test_add", "labels": [{"name": "story", "value": "加法测试案例"}, {"name": "feature", "value": "测试计算器类"}, {"name": "tag", "value": "run(order=1)"}, {"name": "suite", "value": "test_calc_case"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-DH8SI2H"}, {"name": "thread", "value": "32452-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc_case"}]}